services/src/pro/contexts/postgres.rs

```diff
@@ -38,6 +46,8 @@ where
     user_db: Db<PostgresUserDb<Tls>>,
     project_db: Db<PostgresProjectDb<Tls>>,
     workflow_registry: Db<PostgresWorkflowRegistry<Tls>>,
+    dataset_db: Db<ProHashMapDatasetDb>, // TODO: implement postgres version
```

Comment:
This means you cannot restore uploaded datasets?
I'm not sure if we don't need this :thinking:

Reply:
yes, the task read minimum viable to store workflows and that works. I guesstimate it's 20x more work to get persist the datasets also

Reply:
Ok, so then we should add this as a new task to the sprint log.

Reply:
https://trello.com/c/cTGGU4bs